templates/home/index.html.twig

```diff
@@ -4,4 +4,109 @@
 
 {% block body %}
     <div class="nfq-academy">{{ someVariable }}</div>
+    <ul class="list-group m-4">
+        <li class="list-group-item list-group-item-info">Studentai</li>
+        {% for key, value in data %}
+            {% for student in value.students %}
+                <li class="list-group-item">
+                    <a href="/student?name={{ student }}&project={{ key|url_encode }}">{{ student }}</a>
+                    (<span class="badge">Mentorius</span> {{ value.mentors.0 }})
+                </li>
+            {% endfor %}
+        {% endfor %}
+    </ul>
+    <div class="m-4">
+        <form action="/form" method="get">
+            <div class="input-group">
+                <input name="name" type="text" class="form-control" placeholder="Studentas"/>
+            </div>
+            <div class="input-group">
+                <input name="project" type="text" class="form-control" placeholder="Projektas"/>
+            </div>
+            <button type="submit" class="btn btn-success">Sužinoti vertinimą</button>
+        </form>
+    </div>
+    <ul class="list-group m-4">
```

Comment:
Čia tikėjausi, kad panaudosi Twig'o `{{ for` sąlygą, kuri atvaizduos `students.json` duomenis.

Plačiau:
 * https://twig.symfony.com/doc/2.x/tags/for.html
 * (palyginimui) https://www.php.net/manual/en/control-structures.foreach.php
 * https://symfony.com/doc/current/controller.html